.github/workflows/main.yml

```
@@ -0,0 +1,33 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - uses: pnpm/action-setup@v2
+        id: pnpm-install
+        with:
+          version: 7
+
+      - run: pnpm install
+
+      - run: pnpm format:check
+
+      - run: pnpm format:check
```

Duplicated command/should be linting instead

```suggestion
      - run: pnpm format:check

      - run: pnpm lint
```